.github/workflows/shell-app-ios.yml

```diff
@@ -6,6 +6,9 @@ on:
       upload:
         description: 'type "upload" to confirm upload to S3'
         required: false
+      versioned:
+        description: 'Create shell workspace from newest SDK versioned environment'
+        required: false
```

Comment:
for ios shell app to create workspace, classic xdl will install required packages from latest sdk.
in our case, it will install `react-native@https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz`
that's why to add packagesToInstallWhenEjecting override support for classic xdl in this commit: https://github.com/expo/xdl/commit/265186257fb7679c92a4cf9523a311cfb7c78202